### DISCUSSION TITLE
Stars

### DISCUSSION THREAD

**Discussion by diyaryvzz** [community/community#62387] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How and where can I see the stars other people have given to my projects? 
I would appreciate your help.

**Comment by crizmo** [comment]
On the https://github.com/diyaryvzz?tab=repositories repositories tab of your profile you will see star icon on each repo if they have any stars 

![image](https://github.com/community/community/assets/83665497/e7f95d96-b76f-49f5-a861-cd1f2fd3d261)

like lets take on of your repo as an example 
so if you click on the star icon you can see who all the people who have stared your repository

or you can simply go to 
https://github.com/diyaryvzz/ChatGPT-web-sitesi/stargazers

basically :- https://github.com/<github_username>/<repository_name>/stargazers

**Reply by diyaryvzz** [reply]
Thank you for the information.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
